src/main/java/Event.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Implements constructor methods for Event.
+ */
+public class Event extends Task {
+    protected Date eventDate;
+    protected Timing eventTime;
+
+    /**
+     * Instantiates Event object.
+     * @param description Description of event command.
+     * @param eventTime Time of event.
+     */
+    public Event(String description, String eventTime) {
+        super(description, TaskType.EVENT);
+        this.eventTime = new Timing(eventTime);
+        Task.totalTasks++;
+    }
+
+    /**
+     * Constructor to instantiate Event object.
+     * @param description Description of event command.
+     * @param eventDate Date of event.
+     * @param eventTime Time of event.
+     */
```

Comment:
A similar issue to the Deadline constructors I commented on a while back. It would be great if your comments could be more detailed and diverse for constructors of the same class!